moto/iot/models.py

```diff
@@ -2475,5 +2626,63 @@ def update_indexing_configuration(
             thingIndexingConfiguration, thingGroupIndexingConfiguration
         )
 
+    def create_job_template(
+        self,
+        job_template_id: str,
+        document_source: str,
+        document: str,
+        description: str,
+        presigned_url_config: Dict[str, Any],
+        job_executions_rollout_config: Dict[str, Any],
+        abort_config: Dict[str, List[Dict[str, Any]]],
+        job_execution_retry_config: Dict[str, Any],
+        timeout_config: Dict[str, Any],
+    ) -> "FakeJobTemplate":
+        if job_template_id in self.jobs_templates:
+            raise ConflictException(job_template_id)
+
+        job_template = FakeJobTemplate(
+            job_template_id=job_template_id,
+            document_source=document_source,
+            document=document,
+            description=description,
+            presigned_url_config=presigned_url_config,
+            job_executions_rollout_config=job_executions_rollout_config,
+            abort_config=abort_config,
+            job_execution_retry_config=job_execution_retry_config,
+            timeout_config=timeout_config,
+            account_id=self.account_id,
+            region_name=self.region_name,
+        )
+
+        self.jobs_templates[job_template_id] = job_template
+        return job_template
+
+    def list_job_templates(
+        self, max_results: int, current_token: Optional[str]
+    ) -> Tuple[List[Dict[str, Any]], Optional[str]]:
+        all_job_templates = [_.to_dict() for _ in self.jobs_templates.values()]
+
+        if current_token is None:
```

Comment:
Can we use the `@paginate`-decorator to handle pagination? That would simplify the implementation a bit.

See the documentation here: https://docs.getmoto.org/en/latest/docs/contributing/development_tips/utilities.html#paginator

The `list_job_executions_for_thing`-function uses it as well, so you can use that to see it used in practice.

Reply:
Fixed - I was using list_jobs() as an input for my change, so I fixed both methods